src/components/PostEditor.js

```diff
@@ -285,14 +280,14 @@ export class PostEditor extends React.PureComponent {
 
   render () {
     const {
-      post, postEdit, dispatch, imagePending, saving, id, defaultTags, placeholder
+      parentPost, post, postEdit, dispatch, imagePending, saving, id, defaultTags, placeholder
     } = this.props
     const { currentUser } = this.context
     const { description, community_ids, tag, linkPreview } = postEdit
     const selectableTags = uniq(compact([this.props.tag, tag].concat(defaultTags)))
     const { name, showDetails } = this.state
     const editorType = this.editorType()
-    const shouldSelectTag = editorType !== 'project'
+    const shouldSelectTag = !includes(['project', 'event'], editorType)
```

Comment:
hmm. i mean, i don't see any logical reason why they shouldn't, now that they no longer have special auto-populating tag behavior, but i suppose we can leave it off and see if anyone misses it. maybe one less field to look at will lead to higher completion rates.